src/sagas/certificate.js

```diff
@@ -26,6 +26,7 @@ export function* verifyCertificate() {
 
     const certificate = yield select(getCertificate);
     const verificationStatus = yield verifyDocument(certificate);
+    console.log(verificationStatus);
```

Comment:
rm

Reply:
oh no i thought i removed it :/ no git add